src/sys/derivs.cxx

```diff
@@ -327,7 +328,8 @@ const Field2D D2DYDZ(const Field2D &f, CELL_LOC UNUSED(outloc),
   return tmp;
 }
 
-const Field3D D2DYDZ(const Field3D &f, CELL_LOC outloc, DIFF_METHOD method, REGION UNUSED(region)) {
+const Field3D D2DYDZ(const Field3D &f, CELL_LOC outloc, MAYBE_UNUSED(DIFF_METHOD method),
+                     REGION UNUSED(region)) {
```

Comment:
Is the fact that `region` is not used a bug? Seems like this might be surprising.

Reply:
There is a commented out section that begins `// TODO: use region aware implementation`. I'm not sure why it's not used -- @d7919 any ideas?

Reply:
I think this may have just been waiting for the `SpecificInd` offsetting.

Reply:
Ok, we can fix that in a separate PR